doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -472,3 +453,178 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
+
+.. list-table:: Emulated PCI Device Types
+   :header-rows: 1
+   :widths: 20 80
+
+   * - PCI Device Type string
+     - Description
+
+   * - ``xhci``
+     - USB controller used to support USB 3.0 devices, (also supports USB 2.0
+       and USB 1.0 devices).  Parameter ``<bus number>-<port number>`` should be
+       added. The physical USB device attached on the specified bus and port
+       will be detected by User VM and used as expected, e.g., ``xhci,1-2,2-2``.
+
+   * - ``lpc``
+     - Low Pin Count (LPC) bus is used to connect low speed devices to the CPU,
+       for example a serial port, keyboard, or mouse.
+
+   * - ``igd-lpc``
+     - Windows graphic driver requires this virtualized LPC device to complete
+       the display function.
+
+   * - ``ivshmem``
+     - Inter-VM shared memory (ivshmem) virtualized PCI device used specifically
+       for shared memory between VMs. Parameters should be added with the format
+       ``ivshmem,<shm_name>,<shm_size>``. ``<shm-name>`` specifies a shared memory
+       name, and must be listed in :option:`hv.FEATURES.IVSHMEM.IVSHMEM_REGION`
+       as configured using the ACRN configurator tool UI, and needs to start
+       with a ``dm:/`` prefix.
+
+   * - ``ahci``
+     - Advanced Host Controller Interface provides advanced features to access
+       Serial ATA (SATA) storage devices, such as a hard disk. Parameter
+       ``<type:><filepath>*`` should be added: ``type`` could be
+       ``hd`` (harddisk) or ``cd`` (CD-ROM). ``<filepath>`` is the path for the
+       backend file and could be a partition name or a regular file, e.g.,
+       ``ahci,hd:/dev/sda``.
+
+   * - ``ahci-hd``
+     - This is an alias for ``ahci``.
+
+   * - ``ahci-cd``
+     - Advanced Host Controller Interface used to connect with AT Attachment
+       Packet Interface device (for CD-ROM emulation). ``ahci-cd`` has the same
+       parameters with ``ahci``.
+
+   * - ``amd_hostbridge``
+     - Virtualized PCI AMD hostbridge
+
+   * - ``hostbridge``
+     - Virtualized PCI hostbridge, a hardware bridge between the CPU's
+       high-speed system local bus and the Peripheral Component Interconnect
+       (PCI) bus.
+
+   * - ``virtio-blk``
+     - Virtio block type device, a string could be appended to specify the disk
+       image, e.g., ``virtio-blk,/root/test.img``.  You can also could use
+       ``nodisk`` to create a virtio-blk device with a dummy backend.
+
+   * - ``virtio-coreu``
+     - Used for Protected Audio Visual Path (PAVP) session management to provide
+       a User VM with Protected Audio Visual Path service.
+
+   * - ``virtio-input``
+     - Virtio type device to emulate input device. ``evdev`` char device node
+       should be appended, e.g., ``-s
+       n,virtio-input,/dev/input/eventX[,serial]``. ``serial`` is an optional
+       string used as the Uniq of guest virtio input device.
```

Comment:
what is a ``Uniq`` ?

Reply:
unique identification code for the device (if device has it) ,seen in https://github.com/torvalds/linux/blob/master/include/linux/input.h line 49 
updated in doc